homeassistant/components/plex/sensor.py

```diff
@@ -116,8 +116,9 @@ def sync_io_attributes(session):
                 # "picture_of_last_summer_camp (2015)"
                 # "The Incredible Hulk (2008)"
                 now_playing_title = sess.title
-                if sess.year is not None:
-                    now_playing_title += f" ({sess.year})"
+                year = await self.hass.async_add_executor_job(getattr, sess, "year")
```

Comment:
Damn that's dangerous. It's like just this property? All other properties are fine?

Reply:
Yeah, the library lazy loads some of the attributes. I've gone through the rest and all the others seem to be safe to access from async.